usautobuild/actions/good_files.py

```diff
@@ -0,0 +1,120 @@
+import json
+import re
+import shutil
+import time
+import os
+
+from logging import getLogger
+from pathlib import Path
+
+import humanize
+
+from usautobuild.config import Config
+from usautobuild.exceptions import (
+    BuildFailedError,
+    InvalidProjectPathError,
+    MissingLicenseFileError
+)
+from usautobuild.utils import git_version, run_process_shell
+
+log = getLogger("usautobuild")
+
+
+class good_files:
+    def __init__(self, config):
+        self.config = config
+        self.files_to_keep_in_managed = self.get_files_to_keep_in_managed()
+
+    def get_files_to_keep_in_managed(self):
+        path = Path.cwd() / "local_repo/Tools/CodeScanning/CodeScan/CodeScan/bin/Debug/net7.0/FilesToMoveToManaged.json"
+        with open(path, 'r') as file:
+            return json.load(file)
+
+    def make_good_files_build(self, VersionTag) -> None:
```

Comment:
```suggestion
    def make_good_files_build(self, version_tag: str) -> None:
```

add the type hint and respect naming convention. I first thought this was a type